runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -502,6 +540,45 @@ pub fn run(mut config: Config, only_compile: bool) -> RuntimeConfig {
 
     // Measure the speed of all extern function calls.
     for (metric, method_name) in v {
+        // Before read and remove key value, insert key value to all accounts
+        if metric == Metric::storage_read_10b_key_10b_value_1k {
+            testbed = function_call_on_all_accounts(
+                metric,
```

Comment:
The metric should be `warmup`

Reply:
No the sequence of test is:
```
    storage_write_10b_key_10b_value_1k => storage_write_10b_key_10b_value_1k,
    storage_has_key_10b_key_10b_value_1k => storage_has_key_10b_key_10b_value_1k,
    storage_read_10b_key_10b_value_1k => storage_read_10b_key_10b_value_1k,
...
    storage_write_10kib_key_10b_value_1k => storage_write_10kib_key_10b_value_1k,
    storage_has_key_10kib_key_10b_value_1k => storage_has_key_10kib_key_10b_value_1k,
    storage_read_10kib_key_10b_value_1k => storage_read_10kib_key_10b_value_1k,
...
```
before testing read_key_10b_key_10b_value_1k and we need to have all account has key values. Reason of not doing it in warmup is - we also have storage_read_key_10kib_key_10b_value_1k test. And in storage_has_key_10b_key_10b_value_1k we want to measure a state with some key and without some key